src/views/CheckoutSelectAccount.vue

```diff
@@ -72,32 +72,32 @@ export default class CheckoutSelectAccount extends Vue {
         // TODO Redirect to import/create
 
         const id: string = Math.round(Math.pow(2, 32) * Math.random()).toString(16);
-        const map = new Map<string, AddressInfo>();
-        map.set('NQ09 VF5Y 1PKV MRM4 5LE1 55KV P6R2 GXYJ XYQF', new AddressInfo(
+        const map = new Map<string, AccountInfo>();
+        map.set('NQ09 VF5Y 1PKV MRM4 5LE1 55KV P6R2 GXYJ XYQF', new AccountInfo(
```

Comment:
Wait, hard-coded address? :boom:

Reply:
This is a demo function which simulates logging in with a new wallet, will be removed when fixing https://github.com/nimiq/accounts/issues/19.